test/k8s/datapath_configuration.go

```diff
@@ -453,168 +437,9 @@ var _ = Describe("K8sDatapathConfig", func() {
-
-			// Check that the src pod can reach the remote host
-			kubectl.ExecPodCmd(randomNamespace, srcPod, helpers.Ping(k8s2IP)).
-				ExpectSuccess("Failed to ping k8s2 host from src pod")
-
-			// Check that the remote host can reach the dst pod
-			kubectl.ExecInHostNetNS(context.TODO(), k8s1NodeName,
-				helpers.CurlFail("http://%s:80/", dstPodIP)).ExpectSuccess("Failed to curl dst pod from k8s1")
-		}
-
-		It("Pod2pod is encrypted in direct-routing mode", func() {
-			deploymentManager.DeployCilium(map[string]string{
-				"tunnel":               "disabled",
-				"autoDirectNodeRoutes": "true",
-				"encryption.enabled":   "true",
-				"encryption.type":      "wireguard",
-				"l7Proxy":              "false",
-			}, DeployCiliumOptionsAndDNS)
-
-			privateIface, err := kubectl.GetPrivateIface(helpers.K8s1)
-			Expect(err).Should(BeNil(), "Cannot determine private iface")
-			testWireguard(privateIface)
-		})
-
-		It("Pod2pod is encrypted in tunneling mode", func() {
-			deploymentManager.DeployCilium(map[string]string{
-				"tunnel":             "vxlan",
-				"encryption.enabled": "true",
-				"encryption.type":    "wireguard",
-				"l7Proxy":            "false",
-			}, DeployCiliumOptionsAndDNS)
-
-			testWireguard("cilium_vxlan")
-		})
-
-		It("Pod2pod is encrypted in tunneling mode with per-endpoint routes", func() {
-			deploymentManager.DeployCilium(map[string]string{
-				"tunnel":                 "vxlan",
-				"endpointRoutes.enabled": "true",
-				"encryption.enabled":     "true",
-				"encryption.type":        "wireguard",
-				"l7Proxy":                "false",
-			}, DeployCiliumOptionsAndDNS)
-
-			testWireguard("cilium_vxlan")
-		})
-
-	})
-
 	SkipContextIf(func() bool {
 		return helpers.RunsOnGKE() || helpers.RunsWithoutKubeProxy() || helpers.RunsOnAKS()
 	}, "Transparent encryption DirectRouting", func() {
-		var privateIface string
-		BeforeAll(func() {
-			Eventually(func() (string, error) {
-				iface, err := kubectl.GetPrivateIface(helpers.K8s1)
-				privateIface = iface
-				return iface, err
-			}, helpers.MidCommandTimeout, time.Second).ShouldNot(BeEmpty(),
-				"Unable to determine private iface")
-		})
-
-		It("Check connectivity with transparent encryption and direct routing", func() {
-			deploymentManager.Deploy(helpers.CiliumNamespace, IPSecSecret)
-			deploymentManager.DeployCilium(map[string]string{
-				"tunnel":                     "disabled",
-				"autoDirectNodeRoutes":       "true",
-				"encryption.enabled":         "true",
-				"encryption.ipsec.interface": privateIface,
-				"devices":                    "",
-				"hostFirewall.enabled":       "false",
-				"kubeProxyReplacement":       "disabled",
-			}, DeployCiliumOptionsAndDNS)
-			Expect(testPodConnectivityAcrossNodes(kubectl)).Should(BeTrue(), "Connectivity test between nodes failed")
-		})
-
 		SkipItIf(helpers.RunsWithoutKubeProxy, "Check connectivity with transparent encryption and direct routing with bpf_host", func() {
```

Comment:
For this one, you'll also need to run the 5.4 CI job in addition to the 4.19 as we have kube-proxy in both AFAIR.

Reply:
Struggling to see why this matter for my PR changes?

Reply:
I meant that as a reaction to your above `/test-1.25-4.19` comment. I might have missed the `/test` above. That is, we'll have to run this test on 5.4 in addition to 4.19 to ensure the changes above didn't affect it.